packages/wpcom.js/src/lib/me.keyring-connection.js

```diff
@@ -1,4 +1,4 @@
-const root = '/me/keyring-connections/';
```

Comment:
As far as I can see we don't actually use this in Calypso, but we probably should replace our implementation with this...

Reply:
It is used though, but I don't think it changes anything. 🤔 

https://github.com/Automattic/wp-calypso/blob/fa6f0c10e676e0142c6aa03ce3de038fe3b40c2e/packages/wpcom.js/src/lib/me.js#L4

https://github.com/Automattic/wp-calypso/blob/fa6f0c10e676e0142c6aa03ce3de038fe3b40c2e/packages/wpcom.js/src/lib/me.js#L160-L168

Reply:
Yeah it's implemented there, but I don't think Me.keyringConnection() is ever called within Calypso. It might be implemented by other uses of the wpcom.js package though.